ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.14-stable-x86-loaded-upgrade-from-4.14.yaml

```diff
@@ -0,0 +1,65 @@
+base_images:
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-loaded-upgrade-414z
+promotion:
+  to:
+  - namespace: ci
+    tag: latest
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: perfscale-rosa-hcp-multiaz-24nodes-stage-loaded-upgrade414
+  cron: 0 8 * * 3
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: stable
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
```

Comment:
COMPUTE_MACHINE_TYPE: m5.2xlarge instead of default, m5.xlarge, that we use for workloads testing?

Hmm.. looks like we use default for https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-candidate-x86-loaded-upgrade-from-4.12.yaml 
But we introduced m5.2xlarge here https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml#L41 (which I don't remember why or if it was an overlook)

Reply:
Should we try with m5.xlarge and see how it does? And if we are satisfied, then update https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml#L41  to m5.xlarge as well?

Reply:
most of change due to cluster-density-v2 failure, need to increase more cpu/ram to fix the failure, but we can try to use m5.xlarge because of we reduce ITERATION_MULTIPLIER_ENV to 6.

Reply:
I will update the instance type to m5.xlarge in my APF PR, let's wait for your triggered job complete.

Reply:
@liqcui I will go ahead and approve this PR as this looks good overall. Couple of things/reminders for us:
1. Update APF PR https://github.com/openshift/release/pull/47677 for both https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml#L41 to m5.xlarge and 
this new hcp loaded upgrade we are adding.
2. Can we also track below grep error being reported in Jira somewhere (if it's not already done). I want to make sure no issues are masked beneath this grep error. Error snippet from https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/45646/rehearse-45646-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade413/1747432684147183616/artifacts/perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade413/openshift-qe-rosa-upgrade-postcheck/build-log.txt

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~Abnormal co details~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reply:
Sure, I will fix the grep issue, it also happens on Jenkins job, it's not dedicated on Prow CI, no JIRA ticket for this now!